cg/cli/demultiplex/finish.py

```diff
@@ -55,6 +56,16 @@ def finish_flow_cell(
     )
 
 
+@finish_group.command(name="temporary")
+@click.argument("flow-cell-name")
+@click.pass_obj
+def populate_new_metrics(context: CGConfig, flow_cell_name: str):
```

Comment:
More descriptive name here, maybe:
```suggestion
def store_sequencing_metrics(context: CGConfig, flow_cell_name: str):
```

Reply:
This name is also temporary - we will move this command into the finish flow cell command just above once we validated the database entries. So this populate_new_metrics() will be removed!